cfgov/apache/conf.d/headers.conf

```diff
@@ -5,11 +5,16 @@ LimitRequestFieldSize 8190
 KeepAlive On
 MaxKeepAliveRequests 500
 
-
 Header always set X-Frame-Options SAMEORIGIN
 Header always set X-XSS-Protection: "1; mode=block"
 Header always set X-Content-Type-Options: nosniff
 
+# Add LimitRequestBody if the value is provided in the LIMIT_REQUEST_BODY 
+# variable.
+<If "-n osenv('LIMIT_REQUEST_BODY')">
+    LimitRequestBody "${LIMIT_REQUEST_BODY}" 
```

Comment:
I don't know _why_ we use gigantic 8-space indentation in similar blocks below, but we do. Should we do the same here for consistency?

Reply:
Probably, yes.